src/index.html

```diff
@@ -18,15 +20,14 @@
     </head>
     <body  ng-app="Front-Rails">
       <header>
-        <a href="#/question">HERE</a>
-        <a href="#/" class="logo">LOGO!</a>
-        <form class="search">
```

Comment:
The JS is using class="search" to trigger the toggleClass "active" on the buttons and the dropdowns.